# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: docshell/base/nsIContentViewer.idl
# Commit: 4b76024db21c
# Full Hash: 4b76024db21cff006a7424a35c412c6e0a153eb0
# Author: Kris Maglione <maglione.k@gmail.com>
# Date: 2020-09-22 03:47:42
# Regressor Bug: 1655866
# File Overlap Count: 1
# Description:
#   Bug 1655866: Part 5 - Use native PermitUnload implementation from front-end code. r=nika,mconley
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D88318
# ==============================================================================

diff -r 55fb069319f1 -r 4b76024db21c docshell/base/nsIContentViewer.idl
--- a/docshell/base/nsIContentViewer.idl	Mon Sep 21 22:41:01 2020 +0000
+++ b/docshell/base/nsIContentViewer.idl	Mon Sep 21 22:41:04 2020 +0000
@@ -59,6 +59,8 @@
    * eDontPromptAndDontUnload: Don't prompt and return false (unload not permitted)
    *                           if the document (or its children) asks us to prompt.
    * eDontPromptAndUnload: Don't prompt and return true (unload permitted) no matter what.
+   *
+   * NOTE: Keep this in sync with PermitUnloadAction in WindowGlobalActors.webidl.
    */
   cenum PermitUnloadAction : 8 {
     ePrompt = 0,